Cisco/iOS-XR/snmp.conf

```diff
@@ -1,2 +1,10 @@
-! making sure ifIndex gets persisted with SNMP after router reboot 
+! SNMP IOS-XR
+!
+! Set SNMP Access ACL
+ipv4 access-list 33
+32 remark NNS SNMP Access
+33 permit ipv4 209.50.158.0 0.0.1.255 any
+! Set SNMP Acess and Community
+snmp-server community <SNMP_COMMUNITY> RO IPv4 33
```

Comment:
In this line, using `{{snmp_community}}` will leverage templating and auto-replace with what is inserted in the portal UI at device provisioning time - this isn't leveraged in the current UI, but will be in the forthcoming device provisioning UI which will suggest configuration snippets directly related to the values entered in the device provisioning web form. (the Juniper templates already leverage that `{{}}` notation.

Reply:
Oh wow, cool. I was wondering why you all chose to use double {{}}.

Will do!